lib/src/screens/wallet_list/wallet_list_page.dart

```diff
@@ -270,18 +290,23 @@ class WalletListBodyState extends State<WalletListBody> {
 
       if (confirmed) {
         try {
-          auth.changeProcessText(
-              S.of(context).wallet_list_removing_wallet(wallet.name));
+        auth != null ? 
+        auth.changeProcessText(
+          S.of(context).wallet_list_removing_wallet(wallet.name)) 
+          : changeProcessText( S.of(context).wallet_list_removing_wallet(wallet.name));
           await widget.walletListViewModel.remove(wallet);
         } catch (e) {
-          auth.changeProcessText(S
-              .of(context)
-              .wallet_list_failed_to_remove(wallet.name, e.toString()));
+          auth != null ? 
+          auth.changeProcessText(
+            S.of(context).wallet_list_failed_to_remove(wallet.name, e.toString()),
+          ) 
+          : changeProcessText(
+            S.of(context).wallet_list_failed_to_remove(wallet.name, e.toString()),
+          );
```

Comment:
Move `?` to same line as branch for true case. Add left offset for ternary operator cases.